src/components/form/examples/form-span-fields.tsx

```diff
@@ -0,0 +1,215 @@
+ * should span through the columns of the grid.
+ *
+ * Thus the `span` value can be set to `2`, `3`, `4`, `5`, or `all`.
+ * Since we do not offer a *form layout* that has more than five columns,
+ * values higher than 5 (or higher than maximum number of columns in the form)
+ * will only force the file to be full-width, just like what `all` does.
+ *
+ * ````ts
+ * export const schema = {
+ *     …
+ *     properties: {
+ *         name: {
+ *             type: 'string',
+ *             title: 'Comment',
+ *             lime: {
+ *                 layout: {
+ *                       colSpan: 'all',
+ *                   },
+ *               },
+ *         },
+ *         …
+ *     },
+ *     …
+ * };
+ * ````
+ *
+ * ###### Dense layout (Auto reorder fields to avoid empty cells)
+ * Order of fields and number of columns that a field must span through, can
+ * consequently affect the layout of your responsive form when the screen width
+ * changes.
+ * Let's say you have a form with a 4 columned layout, and you specify that its
+ * second field must span (or stretch) through 3 columns.
+ * Now if the container's width decreases, it will forces the form to render its
+ * layout in 3 columns instead. Therefore, the second field has to jump
+ * to the next line to still be able to stretch through 3 columns.
+ * This will leave 2 empty cells in the first row, right after the first field.
+ *
+ * To avoid having these empty cells in the UI, limel-form will automatically
+ * bring up two other fields that span only 1 column, and places them in the
+ * available empty cells in the first row.
+ *
+ * However, you can disable this functionality by setting `dense` to fall in the
+ * options for the grid layout
+ *
+ * ```ts
+ * export const schema = {
+ *     type: 'object',
+ *     lime: {
+ *         layout: {
+ *             type: 'grid',
+ *             dense: false,
+ *         },
+ *     },
+ * };
+ * ```
+ *
+ * :::note
+ * Sometimes order of fields are important in the way users perceive the form.
+ * If you choose to have the default auto-reordering behavior of the forms,
+ * make sure to test yout forms layout in different screen sizes to see whether
+ * you can mitigate unwanted layout changes.
+ *
+ * You can avoid unwanted results by changing order of fields so that they render
+ * appropriately on different screens, or by dividing the form into more
+ * sections.
+ * :::
+ *
+ * ###### Stretching a field vertically
+ * All standard elements that can be used in forms such as `limel-input`,
+ * `limel-select`, `limel-slider`, etc. have a fixed UI in the horizontal axis
+ * and therefore it does not really make sense to stretch them in height,
+ * and we strongly recommend you not to!
+ *
+ * But there are some exceptions. One of them is the `textarea`.
+ * It's always recommended to assign at least two rows for a textarea, to improve
+ * its usability. Since this is a component that is supposed to show multi-lined
+ * pieces of text.
```

Comment:
```suggestion
 * Most standard elements that can be used in forms, such as `limel-input`,
 * `limel-select`, `limel-slider`, etc, have a fixed height, and therefore
 * it does not really make sense to stretch them vertically, and we strongly
 * recommend you not to!
 *
 * But there are some exceptions. One of them is `limel-input-field` with `type='textarea'`.
 * It's always recommended to assign at least two rows for a textarea, to improve
 * its usability. Since this is a component that is supposed to show multi-lined
 * pieces of text.
```